app/components/sac_section_membership_configs/valid_from_selector_component.html.haml

```diff
@@ -0,0 +1,6 @@
+%ul.pagination.pagination-sm
+  - @available_configs.each do |config|
+    %li.page-item{ class: [active?(config) && 'active'] }
```

Comment:
Bin gegen conditionals im template von einer Komponente, ich würde eine `item_class` methode in der Komponente besser finden